src/test/shared/credentials/loginManager.test.ts

```diff
@@ -61,117 +65,86 @@ describe('LoginManager', async function () {
         sandbox.restore()
     })
 
+    function assertTelemetry(expected: Parameters<typeof recordAwsValidateCredentials>[0]): void {
+        assert.deepStrictEqual(recordAwsValidateCredentialsSpy.args[0][0], expected)
```

Comment:
eventually would like tests to actually query the telemetry queue in `src/test/fake/fakeTelemetryService.ts` . Don't really need mocks. Then we can also eliminate the `recordAwsSetCredentialsFn` param

Reply:
Yup agreed. Thought about just ripping up the tests but didn't feel right without an alternative.